app/models/invenio_rdm_record_converter.rb

```diff
@@ -253,7 +253,7 @@ def build_creator_contributor_json(creator)
 
       if dh_user&.orcid.present?
         identifiers = [{"scheme": "orcid", "identifier": dh_user.orcid.split('/').pop}]
-        creatibutor_json[:person_or_org].merge!({"identifiers": identifiers})
+        creatibutor_json.with_indifferent_access[:person_or_org].merge!({"identifiers": identifiers})
```

Comment:
Hashes have been causing annoying issues due to keys as symbols or strings. This is ugly, but based on my testing in the terminal it will get return what we want and work for symbols or keys hopefully resolving this issue for good.

Reply:
ugh sorry you have to go through this with Ruby :grimacing: